hadoop-project/pom.xml

```diff
@@ -921,6 +921,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
```

Comment:
should have a comment above to explain, maybe something on L72 to say why we've had problems with jackson updates

Reply:
I'll add some comments and recheck the dependency scopes.